.github/workflows/winget-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Submit Winget Package to Windows Package Manager Community Repository
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Specific tag name"
```

Comment:
"Specific tag name" ?

Reply:
ha right, we are supposed to run this action manually once the release is done, is that correct ?

Reply:
ok, we can do that but the trigger release above will also trigger it. got it.

Reply:
exact, just understood that! - that action was adapted from another repo. 
By the way, the tag has to be required, updating PR.